include/deal.II/grid/grid_generator.h

```diff
@@ -506,9 +506,9 @@ namespace GridGenerator
    * computed adaptively such that the resulting elements have the least
    * aspect ratio.
    *
-   * If colorize is set to true, the inner, outer, left, and right boundary
-   * get indicator 0, 1, 2, and 3, respectively. Otherwise all indicators are
-   * set to 0.
+   * If colorize is set to true, the inner, outer and equatorial (i.e. where 
+   * the first component is zero) boundary get indicator 0, 1, and 2, respectively. 
```

Comment:
I think the term we were apparently both looking for that would have fit is "equatorial plane". Except, of course, that is almost universally considered the plane where `z==0` (or in 2d, `y==0`). So maybe *any* reference to the word "equator" is confusing here. Maybe just say "inner, outer, and the part of the boundary where $x=0$, get indicator 0, 1, and 2, respectively."?